services/ui-src/src/verbiage/pages/wp/wp-alerts.ts

```diff
@@ -2,7 +2,22 @@ export default {
   initiative: {
     title:
       "This alert will disappear once you add initiatives that meet the topic requirements",
-    description:
-      'Initiatives must meet the following requirements at least once across all the initiatives: <ul><li>Transitions and transition coordination services</li><li>Housing-related supports</li><li>Quality measurement and improvement</li><li>Self-direction (If applicable)</li><li>Tribal (if applicable)</li></ul>To correct this, ensure you answer the 2 questions at the bottom of "State or Territory-Specific Initiatives Instructions" (the previous page), and add at least one initiative for each of these topics on this screen, below. This alert will disappear once they’re met. This alert will reappear if you closed out the only initiative of a required topic.',
+    description: [
+      {
+        type: "p",
+        content:
+          "Initiatives must meet the following requirements at least once across all the initiatives:",
+      },
+      {
+        type: "ul",
+        content:
+          "<li>Transitions and transition coordination services</li><li>Housing-related supports</li><li>Quality measurement and improvement</li><li>Self-direction (if applicable)</li><li>Tribal (if applicable)</li>",
```

Comment:
I like the `children` pattern for content like this because I think it's easier to read. Example here: https://github.com/Enterprise-CMCS/macpro-mdct-mfp/blob/main/services/app-api/forms/wp.json#L228-L232

What do you think?